graphql_compiler/compiler/sqlalchemy_extensions.py

```diff
@@ -64,15 +66,32 @@ def print_sqlalchemy_query_string(query, dialect):
     Returns:
         string that can be ran using sqlalchemy.sql.text(result)
     """
+    supported_dialects = (PGDialect_psycopg2, MSDialect_pyodbc)
+    if isinstance(dialect, supported_dialects):
+        class BindparamCompiler(dialect.statement_compiler):
+            def visit_bindparam(self, bindparam, **kwargs):
+                # A bound parameter with name param is represented as ":param". However,
+                # if the parameter is expanding (list-valued) it is represented as
+                # "([EXPANDING_param])" by default. This is an internal sqlalchemy
+                # representation that is not understood by databases, so we explicitly
+                # make sure to print it as ":param".
+                bindparam.expanding = False
+                return f":{bindparam.key}"
```

Comment:
This is not copy-paste. Notice the explicit string formatting here.

Calling the super method only works sometimes with postgres, and I can't figure out the pattern of when it works and when it doesn't. When it doesn't, it prints `BINDPARAM_{key}` instead of just `:{key}`

Reply:
Should we be worried about injection through `bindparam.key` here?

Reply:
Good catch. I'll figure out something.